src/dbnode/storage/index/block.go

```diff
@@ -528,8 +530,8 @@ func (b *block) queryWithSpan(
 		}
 	}
 
-	iter.AddSeries(size)
-	iter.AddDocs(docsCount)
+	iter.AddSeries(size - sizeBefore)
```

Comment:
can we add a test to verify the change?

Reply:
I wasn't sure if I should since it didn't exists before. I will add.

Reply:
@ryanhall07 Added a unit test